src/include/mutex.i

```diff
@@ -32,7 +32,9 @@ __wt_spin_init(WT_SESSION_IMPL *session, WT_SPINLOCK *t, const char *name)
 	WT_UNUSED(name);
 
 	t->lock = 0;
+	t->name = name;
 	t->stat_count_off = t->stat_app_usecs_off = t->stat_int_usecs_off = -1;
+	t->initialized = 1;
```

Comment:
This is just for consistency, so all the `__wt_spin_init` functions do the same thing.